src/ansys/fluent/core/__init__.py

```diff
@@ -71,6 +71,8 @@ def version_info() -> str:
 )
 EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
 
+PYFLUENT_WORKING_DIR = Path(__file__.split("src")[0]).resolve()
```

Comment:
This is the PyFluent source files directory, not the PyFluent working directory.

The current PyFluent working directory (which is the same as the current Python interpreter working directory) is provided by `os.getcwd()`.

I do not believe we need a `PYFLUENT_WORKING_DIR` constant, the value of the current working directory is always provided by `os.getcwd()`.

Is there a reason for this approach with the module constant that I am missing?

Reply:
@raph-luc 

1. `os.getcwd()` gives different values based on the file location from where it is being called.  Therefore we have used a constant to resolve this issue. 

2. `This is the PyFluent source files directory, not the PyFluent working directory.` - No. Fluent writes files in `Path(__file__.split("src")[0]).resolve()` and reads file with basename from this location.

Just verify 1 and 2 and let me know If I missed anything.

Reply:
`fluent_container.py`

![image](https://github.com/ansys/pyfluent/assets/106588300/7fab37e1-4bf0-4e7f-b161-b68cb254f0ac)

`downloads.py`

![image](https://github.com/ansys/pyfluent/assets/106588300/7bedabaf-fd89-4844-a8dc-68f27a464a93)

Reply:
Regarding point 1, `os.getcwd()` specifically returns the current Python working directory, see the docs: "Return a unicode string representing the current working directory". The PyFluent current working directory should not be changing at all during execution, we'd need to specifically call `os.chdir(...)` for that, and we should never do that. The user has full control over the current Python interpreter working directory, and a Python library like PyFluent should typically not change it.

As to point 2, `Path(__file__.split("src")[0]).resolve()` inside `__init__.py` resolves to wherever PyFluent is installed in the environment. For development purposes, there is a `src` folder so this will be the Python source files. In a user install as I said above, it would return something like `PYFLUENT_WORKING_DIR = C:\test\.venv\Lib\site-packages\ansys\fluent\core\__init__.py` which is most certainly not what we want here.

Reply:
@raph-luc 

I have experimented following thing.

pyfluent.PYFLUENT_WORKING_DIR = `Path(__file__.split("src")[0]).resolve()` (defined in __init__.py)
WORKING_DIR  = `os.getcwd()`         (defined in downloads.py)
WORK_DIR =  `os.getcwd()`               (defined in fluent_container .py)

cwd - `D:`

```
(env1) PS D:\> python
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import ansys.fluent.core as pyfluent
>>> pyfluent.PYFLUENT_WORKING_DIR
WindowsPath('D:/pyfluent')
>>> from ansys.fluent.core.examples.downloads import WORKING_DIR
>>> WORKING_DIR
'D:\\'
>>> from ansys.fluent.core.launcher.fluent_container import WORK_DIR
>>> WORK_DIR
'D:\\'
>>>
```

If we run `downloads.py` and fluent_container.py` individually then it gives different results for `os.getcwd()` as follows.

`os.getcwd()`  = `D:\pyfluent\src\ansys\fluent\core\examples` for `downloads.py` 

`os.getcwd()`  = `D:\pyfluent\src\ansys\fluent\core\launcher` for `fluent_container.py` 

So we have verified that in an interpreter `os.getcwd()` gives correct and consistent results independent of inividual file locations.

We will update this now. 

Thank you.

Reply:
Done. Thank you.